w10h02/test/pgdp/teams/ArtemisTest.java

```diff
@@ -0,0 +1,94 @@
+package pgdp.teams;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Set;
+import static pgdp.teams.Lib.*;
+
+import org.junit.jupiter.api.Disabled; // do not remove
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+
+public class ArtemisTest {
+    @Test
+    @DisplayName("computeScores: kleines Beispiel")
+    public void computeScoresSmall() {
+        var lineup = new Lineup(Set.of(jonas, anatoly), Set.of(julian), Set.of(simon));
+
+        assertEquals(65, lineup.getTeamScore());
+        assertEquals(40, lineup.getTeamSkill());
+        assertEquals(25, lineup.getTeamSynergy());
+    }
+
+    @Test
+    @DisplayName("computeScores: großes Beispiel")
+    public void computeScoresLarge() {
+        var lineup = new Lineup(Set.of(eve, hanna), Set.of(sachmi, jasmine), Set.of(enrico, jakob));
+
+        assertEquals(34505, lineup.getTeamScore());
+        assertEquals(34014, lineup.getTeamSkill());
+        assertEquals(491, lineup.getTeamSynergy());
+    }
+
+    @Test
+    @DisplayName("computeOptimalLineup: kleines Beispiel")
+    public void computeOptimalLineupSmall() throws NoSuchFieldException, IllegalAccessException {
+        var lineup = Lineup.computeOptimalLineup(Set.of(eric, nils, felix, thomas), 2, 1, 1);
+
+        var attackers = getField(lineup, "attackers");
+        var defenders = getField(lineup, "defenders");
+        var supporters = getField(lineup, "supporters");
+
+        assertEquals(2, attackers.size());
+        assertEquals(1, defenders.size());
+        assertEquals(1, supporters.size());
+
+        assertTrue(attackers.contains(nils));
+        assertTrue(attackers.contains(eric));
+        assertTrue(defenders.contains(felix));
+        assertTrue(supporters.contains(thomas));
```

Comment:
Set attackersExpected = Set.of(nils, eric);
Set defendersExpected = Set.of(felix);
Set supportersExpected = Set.of(thomas);

assertEquals(attackersExpected, attackers);
assertEquals(defendersExpected, defenders);
assertEquals(supportersExpected, supporters);